vignettes/installation.Rmd

```diff
@@ -33,8 +33,15 @@ remotes::install_cran('azuremlsdk', repos = 'https://cloud.r-project.org/', INST
 ```
```

Comment:
Can you change "Then, you can use the `install_github` function to install the package." to "Then, you can use the `install_cran` function to install the package." on line 25

Reply:
Actually, it should be the opposite. That sentence is describing how to install via github, but the code sample is for cran, so I'll change the code sample to github.

The actual CRAN instructions are right below that line.